### ISSUE TITLE
Facilitate reusability through referencing of parent Translation Template Bundles

### ISSUE THREAD

**Issue by jonherrmann** [interactive-instruments/etf-webapp#142]
Add a `parent`property to the Translation Template Bundles for reusing translations from a parent one.

Changes:
- [x] Dtos
- [x] Data storage: mapping, [queries](https://github.com/interactive-instruments/etf-bsxds/blob/master/src/main/resources/xquery/ExecutableTestSuite-xdb.xquery#L51)
- [x] [Check Report XSL](https://github.com/interactive-instruments/etf-bsxds/blob/master/src/main/resources/xslt/default/TestRun2DefaultReport.xsl#L1433-L1512)
- [x] [ETS Documentation](http://docs.etf-validator.net/v2.0/Developer_manuals/Developing_Executable_Test_Suites.html#message-template-bundles)
- [x] [Schema doc](https://resources.etf-validator.net/schema/v2/doc/basicTypes_xsd.html#TranslationTemplateBundle)

**Comment by thijsbrentjens** [comment]
Thanks for creating this so quickly. I've updated the ETF software on our test server and tested the example. That works fine. But when using the XSL to create the TTB I got a Sax validaton error after deploying on this part:

<etf:parent ref="EID27117afb-11aa-4e45-aa3d-0e1e65bdccb8" xsi:type="loc"/>

Removing the xsi:type="loc" attribute solved it. I'll create a pull request if you like (need to go finish now), but wanted to let you know.
